daprdocs/content/en/developing-applications/ides/visual-studio-code/vscode-dapr-extension.md

```diff
@@ -0,0 +1,96 @@
+---
+type: docs
+title: "Dapr Visual Studio Code extension overview"
+linkTitle: "Dapr extension"
+weight: 10000
+description:  "How to develop and run Dapr applications with the Dapr extension"
+---
+
+
+Dapr offers a *preview* [Dapr Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-dapr) for local development and debugging of your Dapr applications.
+
+<a href="vscode:extension/ms-azuretools.vscode-dapr" class="btn btn-primary" role="button">Open in VSCode</a>
+
+## Features
+
+### Scaffold Dapr tasks 
+
+* Helps scaffold VS Code task.json and launch.json configurations needed to debug your application within the Dapr environment.
+* Requires an already existing launch.json files to be found in the VS Code workspace.
+* Example NodeJS app:
+
+After scaffolding the launch.json should contain similar snippets:
+```json
+        {
+            "type": "pwa-node",
+            "request": "launch",
+            "name": "Launch Program with Dapr",
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
+            "program": "${workspaceFolder}/app.js",
+            "preLaunchTask": "daprd-debug",
+            "postDebugTask": "daprd-down"
+        }
+```
+After scaffolding the task.json should contain similar snippets:
+
+```json
+		{
+			"appId": "nodeapp",
+			"appPort": 3500,
+			"label": "daprd-debug",
+			"type": "daprd"
+		}
+```
```

Comment:
```suggestion
### Scaffold Dapr debugging tasks

The Dapr extension helps you debug your applications with Dapr using Visual Studio Code's [built-in debugging capability](https://code.visualstudio.com/Docs/editor/debugging).

Using the `Dapr: Scaffold Dapr Tasks` [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) operation, you can update your existing `task.json` and `launch.json` files to launch and configure the Dapr sidecar when you begin debugging.

1. Make sure you have a launch configuration set for your app. ([Learn more](https://code.visualstudio.com/Docs/editor/debugging))
1. Open the Command Palette with `Ctrl+Shift+P`
1. Select `Dapr: Scaffold Dapr Tasks`
1. Run your app and the Dapr sidecar with `F5` or via the Run view.
```

Reply:
Agree with Aaron here. This is not about showing the feature this is about "how" to use the VS Core Extension feature. I certainly could not  figure out how to use the debug extension without this.